server/README.md

```diff
@@ -33,17 +33,17 @@ $ ./gradlew docker
 This runs the Scalar DB Server (you need to specify your local configuration file path with `-v` flag):
 
 ```
-$ docker run -v <your local configuration file path>:/scalardb/server/database.properties -d -p 60051:60051 -p 8080:8080 ghcr.io/scalar-labs/scalardb-server:<version>
+$ docker run -v <your local configuration file path>:/scalardb/server/database.properties.tmpl -d -p 60051:60051 -p 8080:8080 ghcr.io/scalar-labs/scalardb-server:<version>
 
 # For DEBUG logging
-$ docker run -v <your local configuration file path>:/scalardb/server/database.properties -e JAVA_OPTS=-Dlog4j.logLevel=DEBUG -d -p 60051:60051 -p 8080:8080 ghcr.io/scalar-labs/scalardb-server:<version>
+$ docker run -v <your local configuration file path>:/scalardb/server/database.properties.tmpl -e JAVA_OPTS=-Dlog4j.logLevel=DEBUG -d -p 60051:60051 -p 8080:8080 ghcr.io/scalar-labs/scalardb-server:<version>
```

Comment:
It looks like you deleted the `log4j.logLevel` system property from `log4j2.properties.tmpl`. So we also need to change here?

Reply:
Yes, that's right. Thank you for the catch!
If we want to get DEBUG log, we need to specify the log level using `SCALAR_DB_LOG_LEVEL` as follows.

```console
$ docker run -d --name test \
>   -v ~/conf/database.properties:/scalardb/server/database.properties.tmpl \
>   -e SCALAR_DB_LOG_LEVEL=DEBUG \
>   -p 60051:60051 -p 8080:8080 \
>   ghcr.io/scalar-labs/scalardb-server:4.0.0-SNAPSHOT
fad17cf437dd835608463f05bda418572031fbb1f043b956a881c111c63da5a9
```
```console
$ docker logs test | tail -n 10
2022-05-25 07:57:35,209 [DEBUG io.netty.buffer.PooledByteBufAllocator] -Dio.netty.allocator.cacheTrimInterval: 8192
2022-05-25 07:57:35,209 [DEBUG io.netty.buffer.PooledByteBufAllocator] -Dio.netty.allocator.cacheTrimIntervalMillis: 0
2022-05-25 07:57:35,209 [DEBUG io.netty.buffer.PooledByteBufAllocator] -Dio.netty.allocator.useCacheForAllThreads: true
2022-05-25 07:57:35,209 [DEBUG io.netty.buffer.PooledByteBufAllocator] -Dio.netty.allocator.maxCachedByteBuffersPerChunk: 1023
2022-05-25 07:57:35,223 [DEBUG io.netty.channel.DefaultChannelId] -Dio.netty.processId: 12 (auto-detected)
2022-05-25 07:57:35,224 [DEBUG io.netty.channel.DefaultChannelId] -Dio.netty.machineId: 02:42:ac:ff:fe:12:00:03 (auto-detected)
2022-05-25 07:57:35,237 [DEBUG io.netty.buffer.ByteBufUtil] -Dio.netty.allocator.type: pooled
2022-05-25 07:57:35,237 [DEBUG io.netty.buffer.ByteBufUtil] -Dio.netty.threadLocalDirectBufferSize: 0
2022-05-25 07:57:35,237 [DEBUG io.netty.buffer.ByteBufUtil] -Dio.netty.maxThreadLocalCharBufferSize: 16384
2022-05-25 07:57:35,244 [INFO  com.scalar.db.server.ScalarDbServer] Scalar DB server started, listening on 60051
```

I fixed in e79ac64db8f94ebc0c0fa34ec0ee1dc7144f7251.